exercises/src/components/exercise/preview.jsx

```diff
@@ -24,13 +20,6 @@ class ExerciseEditingPreview extends React.Component {
     showEdit: PropTypes.bool,
   }
 
-  @computed get isAuthor() {
-    const { exercise } = this.props;
-    return Boolean(
-      exercise.isNew || find(exercise.authors, { user_id: CurrentUser.id })
```

Comment:
Has the 'isNew' case dropped out? Is it needed? What happens when you create a new exercise, does it fall under the "user is author" part of this test?

Reply:
Yep, good q 😁   in the model it only fails if the author doesn't match or if it's `is_vocab`. otherwise it'll return true so the isNew check is no longer needed.

Reply:
actually I think I'll re-add the `isNew` check since we may stop setting the author on new exercises which would cause this to fail.  Good catch!